Solutions/Legacy IOC based Threat Protection/Deprecated Analytic Rules/CadetBlizzard_Jan2022_IOC.yaml

```diff
@@ -75,24 +75,35 @@ query:  |
   | extend timestamp = TimeGenerated, HostCustomEntity = Computer , AccountCustomEntity = UserName, ProcessCustomEntity = tostring(split(Image, '\\', -1)[-1]), FileHashCustomEntity = tostring(Hashes[1]), AlgorithmCustomEntity = "SHA256"
   )
   )
+    | extend AccountName = tostring(split(UserName, "@")[0]), AccountUPNSuffix = tostring(split(UserName, "@")[1])
```

Comment:
Because you are doing a union, you do have to map the field name to a common name that can be referenced in the entityMapping otherwise it will not pick up outputs from the specific outputs of each query.  We want to remove *CustomEntity but we still need to map to a common name.

Same is true for HostName below...